BootloaderCommonPkg/Library/ShellLib/Shell.c

```diff
@@ -111,9 +111,13 @@ Shell (
   BOOLEAN Start;
   UINTN   Index, Index1;
   SHELL   Shell;
+  CONST SHELL_COMMAND **Iter;
 
+  LoadShellCommands ();
+  for (Iter = Commands; *Iter != NULL; Iter++) {
```

Comment:
Platform specific shell commands have been verified also. Currently, ShellExtensionLib is not changed. Its platform specific commands are updated via registration API before executing Shell console.
I am thinking a way to use platform shell commands with ShellLib only(without ShellExtentionLib). Let me optimize Shell related things based on this change.

Reply:
OK, let us merge this and handle the ShellExtentionLib later on.